src/librepgp/stream-dump.cpp

```diff
@@ -2070,6 +2082,18 @@ stream_dump_one_pass_json(pgp_source_t *src, json_object *pkt)
     return RNP_SUCCESS;
 }
 
+static rnp_result_t
+stream_dump_marker_json(pgp_source_t &src, json_object *pkt)
+{
+    rnp_result_t ret = stream_parse_marker(src);
+
+    if (!obj_add_field_json(
+          pkt, "contents", json_object_new_string(ret ? "invalid" : "PGP"))) {
```

Comment:
I suggest to have a marker constant "PGP" and re-use it.

Reply:
Here and below I think we should better stick to hard-typed text, to detect case when marker body constant is changed.